EESSI-pilot-install-software.sh

```diff
@@ -5,34 +5,7 @@
 
 TOPDIR=$(dirname $(realpath $0))
 
-function echo_green() {
-    echo -e "\e[32m$1\e[0m"
-}
-
-function echo_red() {
-    echo -e "\e[31m$1\e[0m"
-}
-
-function echo_yellow() {
-    echo -e "\e[33m$1\e[0m"
-}
-
-function fatal_error() {
-    echo_red "ERROR: $1" >&2
-    exit 1
-}
-
-function check_exit_code {
-    ec=$1
-    ok_msg=$2
-    fail_msg=$3
-
-    if [[ $ec -eq 0 ]]; then
-        echo_green "${ok_msg}"
-    else
-        fatal_error "${fail_msg}"
-    fi
-}
+source utils.sh
```

Comment:
Probably need to ensure we can pick this up from the same directory as the script (so use `$TOPDIR`)